www/docs/constructs/NextjsSite.about.md

```diff
@@ -494,15 +494,16 @@ new NextjsSite(stack, "Site", {
     transform: (plan) => {
       const username = "admin";
       const password = "P@ssw0rd!";
+      const basicAuth = Buffer.from(`${username}:${password}`).toString("base64");
       plan.cloudFrontFunctions.serverCfFunction.injections.push(`
-        var authHeaders = request.headers.authorization;
-        if (!authHeaders || authHeaders.value !== "Basic ${Buffer.from(${username}:${password}).toString('base64')
-      }"}) {
-          statusCode: 401,
-          statusDescription: "Unauthorized",
-          headers: {
-            "www-authenticate": { value: 'Basic realm="Secure Area"' }
-          }
+        if (request?.headers?.authorization?.value !== '${basicAuth}') {
```

Comment:
I guess you lost the "Basic "prefix.
In my (working) case I have this line like this:

```suggestion
        if (request?.headers?.authorization?.value !== 'Basic ${basicAuth}') {
```

Reply:
@eduardokanema

Reply:
Thanks @jayair